scala-package/infer/src/main/scala/org/apache/mxnet/infer/Classifier.scala

```diff
@@ -110,16 +111,20 @@ class Classifier(modelPathPrefix: String,
   : IndexedSeq[IndexedSeq[(String, Float)]] = {
 
     // considering only the first output
-    val predictResultND: NDArray = predictor.predictWithNDArray(input)(0)
+    val predictResultND: NDArray =
```

Comment:
Nit: A comment explaining that we're preemptively copying to the cpu because all the post-processing is happening on the cpu wouldn't hurt.